lib/scan/pma/engine.py

```diff
@@ -18,34 +18,46 @@ def extra_requirements_dict():
+                          '/phpMyAdmin-2.6.2-rc1/', '/phpMyAdmin-2.6.2-beta1/', '/phpMyAdmin-2.6.2-rc1/',
+                          '/phpMyAdmin-2.6.2/', '/phpMyAdmin-2.6.2-pl1/', '/phpMyAdmin-2.6.3/', '/phpMyAdmin-2.6.3-rc1/',
+                          '/phpMyAdmin-2.6.3/', '/phpMyAdmin-2.6.3-pl1/', '/phpMyAdmin-2.6.4-rc1/', '/phpMyAdmin-2.6.4-pl1/',
+                          '/phpMyAdmin-2.6.4-pl2/', '/phpMyAdmin-2.6.4-pl3/', '/phpMyAdmin-2.6.4-pl4/', '/phpMyAdmin-2.6.4/',
+                          '/phpMyAdmin-2.7.0-beta1/', '/phpMyAdmin-2.7.0-rc1/', '/phpMyAdmin-2.7.0-pl1/',
+                          '/phpMyAdmin-2.7.0-pl2/', '/phpMyAdmin-2.7.0/', '/phpMyAdmin-2.8.0-beta1/',
+                          '/phpMyAdmin-2.8.0-rc1/', '/phpMyAdmin-2.8.0-rc2/', '/phpMyAdmin-2.8.0/', '/phpMyAdmin-2.8.0.1/',
+                          '/phpMyAdmin-2.8.0.2/', '/phpMyAdmin-2.8.0.3/', '/phpMyAdmin-2.8.0.4/', '/phpMyAdmin-2.8.1-rc1/',
+                          '/phpMyAdmin-2.8.1/', '/phpMyAdmin-2.8.2/', '/sqlmanager/', '/mysqlmanager/', '/p/m/a/',
+                          '/PMA2005/', '/pma2005/', '/phpmanager/', '/php-myadmin/', '/phpmy-admin/', '/webadmin/',
+                          '/sqlweb/', '/websql/', '/webdb/', '/mysqladmin/', '/mysql-admin/', '/mya/', '/mysql/admin/',
+                          '/mysql/dbadmin/', '/mysql/sqlmanager/', '/mysql/mysqlmanager/', '/phpMyadmin/', '/phpmyAdmin/',
+                          '/phpmyadmin3/', '/phpmyadmin4/', '/2phpmyadmin/', '/phpmy/', '/phppma/', '/db/phpmyadmin/',
+                          '/db/phpMyAdmin/', '/admin/phpmyadmin/', '/admin/phpMyAdmin/', '/admin/sysadmin/',
+                          '/admin/sqladmin/', '/admin/db/', '/admin/web/', '/admin/pMA/', '/mysql/pma/', '/mysql/pMA/',
+                          '/sql/phpmanager/', '/sql/php-myadmin/', '/sql/phpmy-admin/', '/sql/myadmin/', '/sql/webadmin/',
+                          '/sql/sqlweb/', '/sql/websql/', '/sql/webdb/', '/sql/sqladmin/', '/sql/sql-admin/',
+                          '/sql/phpmyadmin2/', '/sql/phpMyAdmin2/', '/sql/phpMyAdmin/', '/db/myadmin/', '/db/webadmin/',
+                          '/db/websql/', '/db/dbadmin/', '/db/db-admin/', '/db/phpmyadmin3/', '/db/phpMyAdmin3/',
+                          '/db/phpMyAdmin-3/', '/administrator/phpmyadmin/', '/administrator/phpMyAdmin/',
+                          '/administrator/pma/', '/administrator/PMA/', '/phpMyAdmin3/', '/phpMyAdmin4/', '/phpMyAdmin-3/',
+                          '/PMA2011/', '/PMA2012/', '/PMA2013/', '/PMA2014/', '/PMA2015/', '/PMA2016/', '/PMA2017/',
+                          '/PMA2018/', '/pma2011/', '/pma2012/', '/pma2013/', '/pma2014/', '/pma2015/', '/pma2016/',
+                          '/pma2017/', '/pma2018/', '/phpmyadmin2011/', '/phpmyadmin2012/', '/phpmyadmin2013/',
+                          '/phpmyadmin2014/', '/phpmyadmin2015/', '/phpmyadmin2016/', '/phpmyadmin2017/', '/phpmyadmin2018/',]
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065955)

Reply:
Fixed the whitespace mistake